README.md

```diff
@@ -312,6 +312,7 @@ That is the basics of building and testing a feature in the Composable Architect
 * Why isn't `Store` thread-safe? <br> Why isn't `send` queued? <br> Why isn't `send` run on the main thread?
   <details>
     <summary>Expand to see answer</summary>
+
```

Comment:
Weirdly, without this new line the code fragments in the paragraph below dont properly format :/